### ISSUE TITLE
 GSdx HW right side of screen issue 

### ISSUE THREAD

**Issue by FlatOutPS2** [PCSX2/pcsx2#1341]
These games have rendering issues on the right side of the screen.
To replicate play the games in hardware modes. Test on native, as some games have scaling issues that can a cause similar effect.

Confirmed in PCSX2 v1.5.0-dev-701:
- [ ] Wakeboarding Unleashed:
- [ ] Dirge Of Cerberus
image: http://wiki.pcsx2.net/index.php/File:Dirge_of_Cerberus_Final_Fantasy_VII_Forum_3.jpg

Unconfirmed:
- [ ] Just League Heroes (http://wiki.pcsx2.net/index.php/Justice_League_Heroes)
- [ ] Battle Assault 3 featuring Gundam Seed (http://wiki.pcsx2.net/index.php/Battle_Assault_3_featuring_Gundam_Seed)
- [ ] Mortal Kombat: Deception (http://wiki.pcsx2.net/index.php/Mortal_Kombat:_Deception)
- [ ] Mortal Kombat: Deception (http://wiki.pcsx2.net/index.php/Mortal_Kombat:_Armageddon)
- [ ] Family Feud (http://wiki.pcsx2.net/index.php/Family_Feud)

**Comment by refractionpcsx2** [comment]
Justice League Heroes - That was a snowblind game, that should be fine now.

**Comment by prafullpcsx2** [comment]
Mortal kombat games are confirmed to have scaling issues causing similar effect.

**Comment by bositman** [comment]
The Mortal Kombat issue goes away if you use a custom resolution instead of scaling, is this the same with the games mentioned in OP?

**Comment by FlatOutPS2** [comment]
> The Mortal Kombat issue goes away if you use a custom resolution instead of scaling, is this the same with the games mentioned in OP?

No. The confirmed games have the issue even at native, because like I said "some games have scaling issues that can a cause similar effect."
It's a completely different issue.

**Comment by gregory38** [comment]
Could `Family Feud` be tested on latest git ?

**Comment by seta-san** [comment]
tokimeki memorial 3
[gs_20210723205745.gs.zip](https://github.com/PCSX2/pcsx2/files/6871777/gs_20210723205745.gs.zip)
![gs_20210723205745](https://user-images.githubusercontent.com/8851653/126854620-09e2eefa-6bac-46b1-80fe-6865ecaa1fa8.png)
